app/views/provider/admin/account/data_exports/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :page_header_title, 'Export' %>
 <%= form_tag provider_admin_account_data_exports_path, :method => :post do %>
-  <div class="dashboard_card round left">
-    <table width="100%">
```

Comment:
This attribute on the element is deprecated, and it's actually not needed, because this style is applied:
```
table:not(.pf-c-table) {
    background-color: #fff;
    width: 100%;
}
```